spec/features/admin/products_spec.rb

```diff
@@ -114,6 +114,82 @@
       expect(page).to have_content "Unit value can't be blank"
     end
   end
+  
+  describe "soft-deleting" do
+    let!(:product1) { create(:simple_product, name: 'a product to keep', supplier: @supplier) }
+
+    context 'a simple product', js: true do
+      let!(:product2) { create(:simple_product, name: 'a product to delete', supplier: @supplier) }
+
+      before do
+        login_as_admin_and_visit spree.admin_products_path
+
+        within "tr#p_#{product2.id}" do
+          accept_alert { page.find("[data-powertip=Remove]").click }
+        end
+      end
+
+      it 'removes it from the product list' do
+        expect(page).not_to have_selector "tr#p_#{product2.id}"
+        expect(page).to have_selector "tr#p_#{product1.id}"
+      end
+    end
+
+    context 'a shipped product', js: true do
```

Comment:
if we really need JS capabilities (have you tried without `js: true`) we can as well move this option to the `describe` above and it'll apply to all.

Reply:
It does need JS capabilities, but I've moved it to the describe block, as suggested. I'm not really sure when JS capabilities are needed... (further reading for me :nerd_face: )